scripts/zones/Arrapago_Reef/DefaultActions.lua

```diff
@@ -5,5 +5,6 @@ return {
     ['qm2']  = { messageSpecial = ID.text.GLITTERING_FRAGMENTS },
     ['qm3']  = { messageSpecial = ID.text.FLUTTERY_OBJECTS },
     ['qm4']  = { messageSpecial = ID.text.STIFLING_STENCH },
+    ['qm6']  = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
```

Comment:
I am ok with leaving this here. I don't think we can module default actions. We could module the `InteractionGlobal.loadDefaultActionsForZone` and create a new module lookup and that will technically work.